Cargo.lock

```diff
@@ -4,19 +4,13 @@ version = 3
 
 [[package]]
 name = "addr2line"
-version = "0.22.0"
+version = "0.24.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "6e4503c46a5c0c7844e948c9a4d6acd9f50cccb4de1c48eb9e291ea17470c678"
+checksum = "dfbe277e56a376000877090da837660b4427aad530e3028d44e0bffe4f89a1c1"
 dependencies = [
  "gimli",
 ]
 
-[[package]]
-name = "adler"
-version = "1.0.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "f26201604c87b1e01bd3d98f8d5d9a8fcbb815e8cedb41ffccbeb4bf593a35fe"
-
```

Comment:
Are these dependency changes required as part of your update, or are these creeping in another way? I would expect a much [narrower diff here](https://gist.github.com/olix0r/40b68edfdc6b335ed1537d4908d8abdf).

To get back to a cogent state: `cargo co main -- Cargo.lock && cargo check`

Reply:
I think a `cargo update` bumped a bunch of the transitive deps in here, reverting them seems to have sorted it out.